lib/util/db.js

```diff
@@ -336,7 +336,15 @@ const postgresErrorToProblem = (x) => {
   if (error.isProblem === true)
     return reject(error);
 
-  if (error.code === '23502') { // not_null_violation
+  console.log('error.code:', error.code);
```

Comment:
I think the linter will complain about this `console.log()`.

Reply:
:+1:  removed